easybuild/easyconfigs/m/Mmg/Mmg-5.6.0-gompi-2021a.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'CMakeMake'
+
+name = 'Mmg'
+version = '5.6.0'
+
+homepage = 'https://www.mmgtools.org/'
+description = """
+Mmg is an open source software for simplicial remeshing. It provides 3 applications and 4 libraries:
+the mmg2d application and the libmmg2d library: adaptation and optimization of a two-dimensional 
+triangulation and generation of a triangulation from a set of points or from given boundary edges
+the mmgs application and the libmmgs library: adaptation and optimization of a surface triangulation
+and isovalue discretization the mmg3d application and the libmmg3d library: adaptation and optimization 
+of a tetrahedral mesh and implicit domain meshing the libmmg library gathering the libmmg2d, 
+libmmgs and libmmg3d libraries."""
+
+toolchain = {'name': 'gompi', 'version': '2021a'}
+
+source_urls = ['https://github.com/MmgTools/mmg/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['bbf9163d65bc6e0f81dd3acc5a51e4a8c47a7fdae849abc26277e01154fe2437']
+
+builddependencies = [
+    ('CMake', '3.20.1'),
+    ('binutils', '2.36.1'),
+]
+
+dependencies = [('SCOTCH', '6.1.0')]
+
+separate_build_dir = True
+
+configopts = '-DUSE_SCOTCH=ON '
+configopts += '-DLIBMMG2D_SHARED=ON -DLIBMMG3D_SHARED=ON -DLIBMMGS_SHARED=ON -DLIBMMG_SHARED=ON '
+
+sanity_check_paths = {
+    'files': ['bin/%s_O3' % x for x in ['mmg2d', 'mmg3d', 'mmgs']] +
+             ['lib/lib%s.%s' % (x, y) for x in ['mmg2d', 'mmg3d', 'mmgs', 'mmg'] for y in ['a', SHLIB_EXT]],
+    'dirs': ['include/mmg/%s' % x for x in ['mmg2d', 'mmg3d', 'mmgs']]
+}
+
```

Comment:
Is there a sensiblte `santicy_check_commands`  that you could set (i.e. a `--version` or something) to check the binaries will start?

Reply:
I got curious and worked on a variant for 2021b, #15622
We can (and should) add a test step, but sanity checks won't work (bad return values)